src/Illuminate/Cache/FileStore.php

```diff
@@ -173,7 +188,7 @@ protected function getPayload($key)
             $expire = substr(
                 $contents = $this->files->get($path, true), 0, 10
             );
-        } catch (Exception $e) {
+        } catch (Throwable $e) {
```

Comment:
This was intentionally not `Throwable`. Please change it back. `Throwable` but not `Exception` means there was an actual mistake in the code which should be fixed, but `Exception` is interpreted as an accounted for failure occurred.